dart/lib/src/isolate_error_integration.dart

```diff
@@ -10,21 +10,22 @@ import 'sentry_options.dart';
 import 'throwable_mechanism.dart';
 
 class IsolateErrorIntegration extends Integration {
-  late RawReceivePort _receivePort;
+  RawReceivePort? _receivePort;
```

Comment:
in Kotlin we can do this: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/is-initialized.html
so it can be `late` and later verified if its init. or not
do we have something similar? @ueman and @HugoSart

Reply:
As far as I know there is no such thing.